src/main/java/com/yahoo/sshd/server/command/AbstractScpCommand.java

```diff
@@ -119,6 +120,14 @@ public void run() {
             helper.doLogging(e, path);
             LOGGER.info("Error in scp command", e);
         } finally {
+            if (null != sshFile) {
+                try {
+                    sshFile.handleClose();
+                } catch (IOException e) {
+                    LOGGER.error("Error when closing the SshFile.", e);
```

Comment:
Wouldn't it be more helpful if your logger also printed the file that there was an error closing?